### ISSUE TITLE
The question about the edge gradient problem existed in IDR

### ISSUE THREAD

**Issue by pean1128** [Kai-46/IRON#22]
Hi, I have read this paper and some questions appear.  I agree with this paragraph that "For this reason, these methods compute biased gradients w.r.t. the weights of the neural SDF that move surface points along camera rays”. It is easy to understand due to the loss function mentioned in IDR. However, you also display Figure.4 to demonstrate your efficiency which immediately confuses me. Because IDR adopts mask loss to train so that the SDF value will be optimized to fit silhouette supervision and won't lead to similar degradation. Can you give me some inspiration or advice?

**Comment by Jarvisss** [comment]
I have the same question. @Kai-46 Could you give some insight on this question?

> Hi, I have read this paper and some questions appear. I agree with this paragraph that "For this reason, these methods compute biased gradients w.r.t. the weights of the neural SDF that move surface points along camera rays”. It is easy to understand due to the loss function mentioned in IDR. However, you also display Figure.4 to demonstrate your efficiency which immediately confuses me. Because IDR adopts mask loss to train so that the SDF value will be optimized to fit silhouette supervision and won't lead to similar degradation. Can you give me some inspiration or advice?

**Comment by Kai-46** [comment]
We assume the inputs do not include object segmentation masks; hence the mask loss in IDR is not applicable.
